bundles/org.openhab.binding.openuv/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -0,0 +1,6 @@
+eclipse.preferences.version=1
```

Comment:
This file was not part of other migrations, but Markus never explained the reasoning for checking in any .settings file, so I'm fine with this one as well.